baremetalpxeserver/baremetalpxeserver/pxeserveragent.py

```
@@ -407,7 +420,9 @@ def ping(self, req):
         # DETECT ROGUE DHCP SERVER
         cmd = json_object.loads(req[http.REQUEST_BODY])
         mac = self._get_mac_address(cmd.dhcpInterface)
-        ret, output = bash_ro("nmap -sU -p67 --script broadcast-dhcp-discover --script-args broadcast-dhcp-discover.mac=%s -e %s | grep 'Server Identifier'" % (mac, cmd.dhcpInterface))
+        ret, output = bash_ro(
```

Ruff (0.8.2)

`bash_ro` may be undefined, or defined from star imports

(F405)